Add Moq.EntityFrameworkCore (#4167)
dotnet/EntityFramework.Docs: entity-framework/core/extensions/index.md

[Website](https://instantwebapi.com/)

> ### Moq.EntityFrameworkCore

Reviewer: This is an extension, not a tool, please move down

This library helps you mock EntityFramework contexts. It allows you to test methods that are using `DbSet<TEntity>` or `DbQuery<TEntity>` from `DbContext` in an effective way.

> For EF Core: 3, 5, 6, 7.

Reviewer: I meant move this up to the sentence above, som it becomes a single paragraph (just like almost all other extensions on this list)

> This library helps you mock EntityFramework contexts. It allows you to test methods that are using `DbSet<TEntity>` or `DbQuery<TEntity>` from `DbContext` in an effective way.
> 
> For EF Core: 3, 5, 6, 7.

Reviewer 2: ```suggestion
This library helps you mock EntityFramework contexts. It allows you to test methods that are using `DbSet<TEntity>` or `DbQuery<TEntity>` from `DbContext` in an effective way. For EF Core: 3, 5, 6, 7.
```

[GitHub repository](https://github.com/R4ND3LL/EntityFrameworkRuler/) | [CLI Tool NuGet](https://www.nuget.org/packages/EntityFrameworkRuler/) | [Design NuGet](https://www.nuget.org/packages/EntityFrameworkRuler.Design/)

### Moq.EntityFrameworkCore

> This library helps you mocking EntityFramework contexts. Now you will be able to test methods that are using `DbSet<TEntity>` or `DbQuery<TEntity>` from `DbContext` in an effective way.

Reviewer 3: ```suggestion
This library helps you mock EntityFramework contexts. It allows you to test methods that are using `DbSet<TEntity>` or `DbQuery<TEntity>` from `DbContext` in an effective way.
```

> For EF Core: 3, 5, 6, 7.

Reviewer: Please move up to the paragraph above - otherwise LGTM

Author: I moved to the paragraph above (Tools). I doubted whether this should be seen as a Tool or an Extension.
